operations_test.go

```diff
@@ -61,12 +64,17 @@ type fakeGNOIClient struct {
 	RebootStatuser func(context.Context, *spb.RebootStatusRequest, ...grpc.CallOption) (*spb.RebootStatusResponse, error)
 	KillProcessor  func(context.Context, *spb.KillProcessRequest, ...grpc.CallOption) (*spb.KillProcessResponse, error)
 	Installer      func(context.Context, ...grpc.CallOption) (ospb.OS_InstallClient, error)
+	SCPer func(ctx context.Context, in *spb.SwitchControlProcessorRequest, opts ...grpc.CallOption) (*spb.SwitchControlProcessorResponse, error)
```

Comment:
Hm, maybe rename this to SwitchController. I'll try to do a refactoring cleanup of all these names separately.